sig-network/role-sig-chair.md

```diff
@@ -0,0 +1,70 @@
+# Role: SIG Chair
+
+## Background
+
+Please see the
+[standard description of a SIG chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair),
+which does a good job describing the expectations of the more “administrative”
+aspect of being a SIG chair.
+
+## What's expected
+
+Rather than repeat the doc linked above, this doc will just add some color.
+
+The minimum effective engagement for a SIG chair is to “run the show” (really
+“make sure the show is running” - delegation is good!).  That generally means
+building an agenda for and running regular meetings, curating our
+community-facing metadata, and acting as liaison to other groups (other SIGs
+and steering, mostly).
```

Comment:
there is also community updates, annual report, enhancement tracking, etc. Is this paragraph summarizing the link https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair or trying to boil it down to "_important_ bare minimum"?

Reply:
maybe the what's expected section can be changed to exclude this section and concentrate on the importance of other things like new issues triage and trends analysis? For big SIGs this is an important addition that is not obvious from the description in the link

Reply:
I wanted to emphasize that there is a "basic" role.  It may play down this aspect, I'll add a few words, burt I don't want to repeat too much.